operator/pkg/central/extensions/common_test.go

```diff
@@ -19,7 +18,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/interceptor"
 )
 
-type secretVerifyFunc func(t *testing.T, data types.SecretDataMap)
+type secretVerifyFunc func(t *testing.T, central *platform.Central, secret *v1.Secret)
```

Comment:
I changed the signature of the `secretVerifyFunc` in various tests because
- Initially the `common/extensions/secret_reconciliator.go` checked that an ownerReference was present
- Now, some secrets (e.g. central-db-password) do not have ownerReferences. 
- I moved the `ownerReference` check out of the `common/extensions/secret_reconciliator` to the various tests that make use of it